docs/book/src/SUMMARY.md

```diff
@@ -2,19 +2,21 @@
 
 [Introduction](./introduction.md)
 
-[Quick Start](./quick-start.md)
+- [User Guide](./user/guide.md)
+    - [Concepts](./user/concepts.md)
+    - [Quick Start](./user/quick-start.md)
```

Comment:
Using the Netlify preview, the link to the quickstart doc from the Introduction is no longer working.

Reply:
Fixed, thanks!